simbad/lattice/tests/test_latticesearch.py

```diff
@@ -148,6 +148,16 @@ def test_check_sg(self):
         reference_data = 'P1'
         
         self.assertEqual(data, reference_data)
+
+    def test_pdb_in_results(self):
+        """Test case for LatticeSearch.pdb_in_results"""
+
+        from simbad.lattice.latticescore import LatticeSearchResult
+        results = []
+        score = LatticeSearchResult("4Z88", "", "", "62.420,62.492,163.64,90.234,90.000,119.962", 4.164, 3.892, 0.272, 23300, 0.1800)
+        results.append(score)
+
+        self.assertTrue(self.LS.pdb_in_results("4Z88", results)) 
         
```

Comment:
Add more test cases, i.e.  one where it returns ``False``. Maybe one with lower/uppercase issues ...